core/src/main/scala/org/apache/spark/util/AccumulatorV2.scala

```diff
@@ -112,7 +112,18 @@ abstract class AccumulatorV2[IN, OUT] extends Serializable {
    * Creates a new copy of this accumulator, which is zero value. i.e. call `isZero` on the copy
    * must return true.
    */
-  def copyAndReset(): AccumulatorV2[IN, OUT]
+  def copyAndReset(): AccumulatorV2[IN, OUT] = copy().reset()
```

Comment:
Is the `copy` only used here? cc @rxin when will the `copy` be useful in other places?